src/docs/development/tools/devtools/performance.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Using the Performance view
+description: Learn how to use the DevTools performance view.
+---
+
+## What is it?
+
+The performance view allows you to record and profile a session from your Dart application.
+
+{{site.alert.note}}
+  **If you are running a Flutter application, use a profile build to analyze performance.**
+  Cpu profiles are not indicative of release performance unless your Flutter application is
+  run in profile mode.
+{{site.alert.end}}
+
+## CPU Profiler
+Start recording a CPU profile by clicking Record. When you are done recording, click Stop. At this
+point, CPU profiling data will be pulled from the VM and displayed in the profiler views (Call Tree,
```

Comment:
will be pulled => is pulled   (use present tense: https://developers.google.com/style/tense)